config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
 
-  root 'instructors#index'
+  root "challenges#index"
```

Comment:
Lets be consistent with our string quotes. We will use single quotes for strings, unless we need to do string interpolation.